bot/extensions/ctf.py

```diff
@@ -18,15 +18,23 @@ class Ctf(commands.Cog):
 
     def __init__(self, bot):
         self.bot = bot
-
+    
     @commands.group()
     async def ctf(self, ctx):
         self.guild = ctx.guild
         self.gid = ctx.guild.id
-
         if ctx.invoked_subcommand is None:
             await ctx.send('Invalid command passed.  Use !help.')
 
+    @ctf.command()
+    async def addcat(self, ctx, *params)
+         try:
+            CHALLENGE_CATEGORIES.extend([params])
```

Comment:
Me to na kamis extend to list enen persintent, kathe fora p kamni restart o zolos en na fevgoun ta alla categories.
Prosthese apla to forensics sta categories an einai.